lib/network/modules/components/nodes/util/CircleImageBase.js

```diff
@@ -14,12 +14,22 @@ class CircleImageBase extends NodeBase {
   }
 
   setImages(imageObj, imageObjAlt) {
-    if (imageObj) {
-      this.imageObj = imageObj;
+    if (!this.selected) {
+      if (imageObj) {
+        this.imageObj = imageObj;
 
-      if (imageObjAlt) {
-        this.imageObjAlt = imageObjAlt;
+        if (imageObjAlt) {
```

Comment:
> There is no hope for TypeScript in vis.js

I understand that it would be an important update but maybe would it bring `vis.js` (or `.ts`??) to the next step! An intermediary step could be to switch to strict ES6 using eslint (with no `var`s etc.)